Anticapitalization Aktion

Sweedle opened the pull request:
:cl: Sweedle
tweak: Changed a ton of random capitalization. RIP Washing Machine.
/:cl:

99% of this is just going through and either: 

- uncapitalizing shit
- adding \improper where it makes sense
- removing \improper where I don't _think_ it makes sense (things that were already not capitalized)

Also trimmed out a couple of really extraneous comments, including one with a bad word! :(

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

Sweedle committed:
Anticapitalization

Changes capitalization in a bunch of stuff.

SierraKomodo commented on code/game/objects/items/weapons/RSF.dm:
```diff
@@ -5,7 +5,7 @@ RSF
 
 /obj/item/rsf
-	name = "\improper Rapid-Service-Fabricator"
+	name = "\improper rapid service fabricator"
```
`\improper` can be removed here.

SierraKomodo commented on code/game/objects/items/weapons/storage/uplink_kits.dm:
```diff
@@ -122,7 +122,7 @@
 
 /obj/item/storage/box/syndie_kit/cigarette
-	name = "\improper Tricky smokes"
+	name = "\improper tricky smokes"
```
`/improper` can be removed here.

SierraKomodo commented on code/modules/mechs/equipment/utility.dm:
```diff
@@ -726,7 +726,7 @@
 
 /obj/item/mech_equipment/ionjets
-	name = "\improper exosuit manouvering unit"
+	name = "exosuit manouvering unit"
```
While we're here, `maneuvering` is the correct spelling.

SierraKomodo commented on code/modules/modular_computers/file_system/programs/command/card.dm:
```diff
@@ -1,6 +1,6 @@
 /datum/computer_file/program/card_mod
 	filename = "cardmod"
-	filedesc = "ID card modification program"
+	filedesc = "ID card Modification Program"
```
If we're capitalizing here, capitalize `Card` too.

Sweedle committed:
Capitalizations


Strips bad word comments

Use better words please (1984 or whatever)
Unfucks typo

please stop runtiming
